analyses/copy_number_consensus_call/run_consensus_call.sh

```diff
@@ -18,3 +18,25 @@ python3 src/scripts/merged_to_individual_files.py \
     --cnvkit ../../data/pbta-cnv-cnvkit.seg.gz \
     --freec ../../data/pbta-cnv-controlfreec.tsv.gz \
     --snake ../../scratch/config_snakemake.yaml
+
+ls ../../scratch/
+cat ../../scratch/config_snakemake.yaml
+
+## Run the Snakemake pipeline
```

Comment:
Maybe here describe here what the snakemake pipeline is doing, in broad strokes.